### ISSUE TITLE
TextureMesh resolution level takes huge impact on runtime

### ISSUE THREAD

**Issue by donlk** [cdcseacave/openMVS#534]
### Specifications like the version of the project, operating system, and hardware
OpenMVS: commit 304bb3cbe5d591d6910ea0842b129b1a0661c512
OS: Debian 10
Hardware: AMD Ryzen 3700X
                  32GB DDR4 3000Mhz RAM
                  NVIDIA GeForce 1070Ti

### Steps to reproduce the problem
Run TextureMesh with --resolution-level greater than 0.

Hey,
Firstly i'd like to say thank you for the your continuous support for OpenMVS, i think it improved greatly in the last couple of years.
I've been experimenting with the different options for TextureMesh trying to improve runtime performance. Among these options was "--resolution-level" which allegedly scales down the input images for faster runtime speed i assume. Thing is, it does just the opposite. Performance sometimes is by an order of magnitude slower when using "--resolution-level" with values greater than 0.
Is this correct behavior?

**Comment by cdcseacave** [comment]
no, it is not, I never seen runtime increasing using a lower resolution
however the bottleneck in texturing is usually too dense mesh, so try decimating the mesh before (make sure it is manifold)

**Comment by cdcseacave** [comment]
Closing as stale. The codebase has been substantially updated on the \develop\ branch. Please reopen with reproduction steps if the issue persists on the latest code.
